app/collections/viewCollectionsCtrl.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+angular
+    .module('Yellr')
+    .controller('viewCollectionsCtrl', ['$scope', '$rootScope', '$location',
+                'collectionApiService',
+    function ($scope, $rootScope, $location, collectionApiService) {
+
+        if ($rootScope.user === undefined) {
+            $location.path('/login');
+            return;
+        }
+
+        $scope.user = $rootScope.user;
+
+        $scope.$parent.clear();
+        $scope.$parent.collectionsPage = true;
+
+        /**
+         * Places all collections in scope
+         *
+         * @return void
+         */
+        $scope.getCollections = function () {
```

Comment:
I've might have messed this up earlier, but normally I'll only reserve `$scope` functions to things that are specifically called through the DOM (using `ng-click` or something). Otherwise I just create a function which looks something like:
```javascript
var _getCollections = function () {
    ...
};
_getCollections();
```